src/components/Dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -0,0 +1,69 @@
+import { useAppList } from 'src/routes/safe/components/Apps/hooks/appList/useAppList'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { Link } from 'react-router-dom'
+import { getSafeAppUrl, SafeRouteParams } from 'src/routes/routes'
+import { useSelector } from 'react-redux'
+import { currentSafe } from 'src/logic/safe/store/selectors'
+import { getShortName } from 'src/config'
+import { ReactElement, useMemo } from 'react'
+import Row from 'src/components/layout/Row'
+import Col from 'src/components/layout/Col'
+import styled from 'styled-components'
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+
+const StyledImage = styled.img`
+  max-width: 64px;
+  max-height: 64px;
+`
+
+const StyledLink = styled(Link)`
+  margin-top: 10px;
+  text-decoration: none;
+`
+
+const StyledRow = styled(Row)`
+  gap: 24px;
+  flex-wrap: inherit;
+`
+
+// TODO: Replace with type from gateway-sdk once available
+type SafeAppWithTags = SafeApp & { tags: string[] }
```

Comment:
It's nice to see you extend the type here but we should do this in the gateway types first.

Reply:
Just saw that its already there in v2.10.3. Will update the dependency and remove this type.